.eslintrc

```diff
@@ -1,6 +1,14 @@
 {
   "extends": "airbnb",
 
+  "plugins": [
+    "react-hooks",
```

Comment:
fwiw you can bring this in with `"extends": ["airbnb", "airbnb/hooks"]` instead, if you're on the latest version of the config